grr/server/grr_response_server/gui/api_plugins/metadata.py

```diff
@@ -37,3 +37,25 @@ def Handle(
     result.revision = version_dict["revision"]
     result.release = version_dict["release"]
     return result
+
+
+class ApiGetOpenApiDescriptionResult(rdf_structs.RDFProtoStruct):
```

Comment:
All classes need to have a docstring (even if it is quite dummy). Otherwise, our internal linter complains.

Reply:
Added it. Thank you and please let me know if it's not ok.